src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/8.diff.sql

```diff
@@ -8,6 +8,42 @@ ADD
 
 GO
 
+DROP PROCEDURE IF EXISTS dbo.UpsertResource;
+
+DROP PROCEDURE IF EXISTS dbo.UpsertResource_2;
+
+DROP TYPE IF EXISTS dbo.ResourceWriteClaimTableType_1;
```

Comment:
I think it's too early to drop these. Consider this scenario:
- Server is running latest version of code (this) and on V7 of the schema. So it uses UpsertResource_2 to insert resources.
- We run 8.diff.
- Then UpsertResource2 is gone before the server knows to use the V8 schema.

Reply:
As discussed, I reverted the changes to drop the types in 8113277 and created the story [AB#80754](https://microsofthealth.visualstudio.com/Health/_workitems/edit/80754) to make sure we drop the unused types in a later schema version.